src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -131,7 +131,7 @@ internal string GetDebuggerDisplay()
                     usingsBinder = new InContainerBinder(binder.Container, binder.Next, imports);
                 }
 
-                var uniqueUsings = PooledHashSet<NamespaceOrTypeSymbol>.GetInstance();
+                var uniqueUsings = new HashSet<NamespaceOrTypeSymbol>(SymbolEqualityComparer.ConsiderEverything);
```

Comment:
lack of pooling seems concerning.  can we not have different pools for the different comparers?

Reply:
Fixed.